libs/books/data-access/src/lib/+state/reading-list.actions.ts

```diff
@@ -41,3 +41,18 @@ export const confirmedRemoveFromReadingList = createAction(
   '[Reading List API] Confirmed remove from list',
   props<{ item: ReadingListItem }>()
 );
+
+export const markBookAsFinished = createAction(
+  '[Reading List API] Mark book as finished',
+  props<{ item: ReadingListItem; finishedDate: string }>()
+);
+
+export const markBookAsFinishedSuccess = createAction(
+  '[Reading List API] Mark book as finished success',
```

Comment:
Same here about the conventions, see line 26 of same file. Again, not that important, but it's nice, shows awareness of conventions etc

Reply:
Updated accordingly